presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -876,4 +897,18 @@ Hive Connector Limitations
 
 * :doc:`/sql/delete` is only supported if the ``WHERE`` clause matches entire partitions.
 * :doc:`/sql/alter-schema` usage fails, since the Hive metastore does not support renaming schemas.
+* Writing to and creation of transactional tables is not supported.
+* Reading ORC ACID tables created with Hive Streaming ingest is not supported.
```

Comment:
streaming

Reply:
i think this is a name

Reply:
Really? Not this one right? https://www.hivestreaming.com/

If it is a name .. lets add a link .. if not.. lets use lowercase

Reply:
This one: https://cwiki.apache.org/confluence/display/Hive/Streaming+Data+Ingest, will link

Reply:
thanks